ote/src/clj/ote/services/service_search.clj

```diff
@@ -118,30 +118,34 @@
      :results results
      :filter-service-count (count ids)}))
 
+(defn- parse-query-parameters [params]
```

Comment:
Not a biggie, but might consider renaming this function.
It is kind of general now. It only parses parameters that are relevant to service search.
And in some occasions we have used to name functions by their return values.
So -> parsed-service-query-parameters. or something similar.

Reply:
Absolutely right... and it has extra verbiage but is a pure function.